en/phones.ftl

```diff
@@ -0,0 +1,157 @@
+phone-onboarding-step4-code-success-subhead-body-p1 = { -brand-name-relay } sent you a text with a new contact card through which we’ll forward your calls and messages.
+phone-onboarding-step4-code-success-subhead-body-p2 = Please save the contact so you can identify your forwarded messages and calls.
+phone-onboarding-step4-code-success-cta = Continue
+phone-onboarding-step4-results= No results found. Please try again.
+
+## Phone Settings
+
+phone-settings-caller-sms-log = Caller and texts log
+phone-settings-caller-sms-log-description = Allow { -brand-name-firefox-relay } to keep a log of your callers and text senders.
+phone-settings-caller-sms-log-warning = If you decide to opt out from this preference, you will lose the ability to block and reply to senders or callers. If you’ve blocked any, they will become unblocked and your existing caller and text sender log will be permanently cleared from your history.
+
+## Phone Resend SMS Banner
+
+phone-banner-resend-welcome-sms-cta = Resend welcome text
+phone-banner-resend-welcome-sms-title = Quick Tip
+phone-banner-resend-welcome-sms-body =  Remember to save the contact we shared with you by text to help you identify messages forwarded by { -brand-name-relay }. Can’t find it?
+
+## Phone What's New
+
+whatsnew-feature-phone-header = Introducing phone number masking
+whatsnew-feature-phone-snippet = With phone number masking, you can now create a phone number mask that helps you…
+whatsnew-feature-phone-description = With phone number masking, you can now create a phone number mask that helps you get texts and calls without revealing your true number.
+whatsnew-feature-phone-upgrade-cta = Upgrade now
+
+## Phone Dashboard
+
+phone-statistics-remaining-call-minutes = Remaining call minutes
+phone-statistics-remaining-texts = Remaining texts
+phone-statistics-calls-texts-forwarded = Calls and texts forwarded
+phone-statistics-calls-texts-blocked = Calls and texts blocked
+phone-dashboard-metadata-forwarded-to = Forwarded to:
+phone-dashboard-metadata-date-created = Date Created:
+phone-dashboard-number-copied = Copied!
+phone-dashboard-forwarding-toggle-enabled = Forwarding enabled
+phone-dashboard-forwarding-toggle-disabled = Forwarding disabled
+phone-dashboard-forwarding-enabled = { -brand-name-relay } is currently forwarding all phone calls and text messages to your true phone number.
+phone-dashboard-forwarding-blocked = { -brand-name-relay } is blocking all phone calls and text messages—you will not receive anything from your phone number mask.
+phone-dashboard-senders-header = Callers and text senders
+phone-dashboard-sender-table-title-sender = Sender
+phone-dashboard-sender-table-title-activity = Latest Activity
+phone-dashboard-sender-table-title-action = Action
+phone-dashboard-sender-disabled-body =  You have disabled the Caller and Sender log. Go to your settings to
+        enable { -brand-name-relay } to keep a log of your callers and senders.
```

Comment:
Is there a reason this string is broken into two lines?

Reply:
Don't think so: 2a33745